src/app/body/board/sprint-details/sprint-details.component.ts

```diff
@@ -94,7 +94,7 @@ export class SprintDetailsComponent implements OnInit {
   }
 
   showDeleted() {
-    this.currentSprint.emit(-2);
+    this.currentSprint.emit(this.currentSprintNumber);
```

Comment:
This will show the tasks of the Active Sprint instead of Deleted Tasks

Reply:
Good point @sanjaykrishna1203